### DISCUSSION TITLE
Next?

### DISCUSSION THREAD

**Discussion by jdonahue94** [smartcontractkit/full-blockchain-solidity-course-py#716] [General]
Hi All,

First of all. I want to thank everyone who has contributed to this tutorial and the subsequent discussions. World class content! I'm wondering of any recommendations for my next weekend study session? Specifically, I'm looking to expand my knowledge of building a Dapp...less DeFi oriented, more social.

Thanks
Jacob

**Comment by cromewar** [comment]
Hello @jdonahue94 sorry for the delay, well there are a lot of thing you can learn. I highly recommend to check out the following:

1. [Moralis](https://moralis.io/) This is a really nice platform for backend on blockchain, check it out their guides.
2. Visit Dapp University channel on youtube.
3. Also give a check to Avalanche blockchain.
